operators/advanced_network/advanced_network/managers/dpdk/adv_network_dpdk_mgr.cpp

```diff
@@ -1958,7 +1934,10 @@ void DpdkMgr::free_tx_metadata(BurstParams* burst) {
 
 Status DpdkMgr::get_tx_metadata_buffer(BurstParams** burst) {
   if (rte_mempool_get(tx_metadata, reinterpret_cast<void**>(burst)) != 0) {
-    HOLOSCAN_LOG_CRITICAL("Failed to get TX meta descriptor");
+    HOLOSCAN_LOG_CRITICAL("Running out of TX meta buffers due to high rates. Either increase "\
+      "your number of metadata buffers (current: {}) with `tx_meta_buffers` (will "\
+      "increase memory usage) or increase your `batch_size` for port {} queue {} (will increase "\
+      "latency)", cfg_.tx_meta_buffers_, (*burst)->hdr.hdr.port_id, (*burst)->hdr.hdr.q_id);
```

Comment:
I think that's going to be very helpful actionable info.

What about for Rx? should we have something similar?

Reply:
It's in the same commit

Reply:
Also, should that message be common to all managers? could we have that log in a base implementation when a manager-specific implementation returns `NO_FREE_BURST_BUFFERS` ?